helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -100,3 +111,28 @@ internal class HelperMain : CliktCommand(name = ORTH_NAME, epilog = "* denotes r
         printStackTrace = stacktrace
     }
 }
+
+/**
+ * Read [ortFile] into an [OrtResult] and return it.
+ */
+fun CliktCommand.readOrtResult(ortFile: File): OrtResult {
```

Comment:
Why are these functions of `CliktCommand`?
Isn't the `CliktCommand` not used?

Reply:
Because we need an object to log an / where the log extension is defined.